mfs/ops.go

```diff
@@ -101,7 +101,7 @@ func PutNode(r *Root, path string, nd ipld.Node) error {
 type MkdirOpts struct {
 	Mkparents bool
 	Flush     bool
-	Prefix    *cid.Prefix
+	Prefix    cid.Builder
```

Comment:
Same. Should probably rename this to `CidBuilder`.

Reply:
Ditto.